### PR TITLE
Removed Permission request from OnDeviceReady to Initialization Succe…

### PR DESCRIPTION
**Pull request by nehabluedot**
Loc permission should be requested from the App after Init Sucess

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,22 +12,24 @@
...
+            }
         },
         "platforms": [
             "ios",
             "android"
         ]
     },
     "devDependencies": {
-        "@bluedot-innovation/cordova-plugin": "^3.0.0",
+        "@bluedot-innovation/cordova-plugin": "file:../PointSDK-Plugin-Cordova",
```
Comment (jsprincep) [inline package.json]: Be sure to revert this hard coded path before merging.

File: www/js/index.js [hunk 2]
```diff
@@ -38,13 +38,6 @@ var app = {
         document.getElementById( "projectId" ).value = projectId;
         document.getElementById( "destinationId" ).value = destinationId;
         document.getElementById( "orderId" ).value = orderId;
-
-        // Request Location When in Use Authorization
-        cordova.plugins.diagnostic.requestLocationAuthorization(
```
Comment (jsprincep) [inline www/js/index.js:43]: Have you tested this change on iOS?

Comment (nehabluedot) [reply www/js/index.js:43]: Yes, Location Permission is asked from `bdFunctions.js`  for `iOS`

File: package.json (hunk 1 above)
Comment (nehabluedot) [reply package.json]: Updated the dependency path with the Cordova plugin git-branch

**Review by jsprincep — APPROVED** [review]
Looking good 👍

**Pull request merged**